### PR TITLE
staging v4.0.0-alpha2

### PR DESCRIPTION
**Pull request by danbryan**
staging v4.0.0-alpha2

### CODE REVIEW

**Review by nourspace — APPROVED** [review]

File: testnet/grand-1/chain-upgrades/v4.0.0-alpha2/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Upgrade to v4.0.0-alpha2
+
+## Overview
+
+We are excited to announce the upcoming software upgrade for the Noble `grand-1` testnet chain. This upgrade will transition us to `v4.0.0-alpha2`. This upgrades adds stability to the cctp and router modules for continued testing.
```
Comment (johnletey) [inline testnet/grand-1/chain-upgrades/v4.0.0-alpha2/README.md:5]:
```suggestion
We are excited to announce the next software upgrade for the Noble `grand-1` testnet chain. This upgrade will transition the network to `v4.0.0-alpha2`. This upgrade adds stability to both the CCTP and Router modules, allowing continued testing.
```

File: testnet/grand-1/chain-upgrades/v4.0.0-alpha2/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+**Key Details:**
+- **Target Block:** [`2682700`](https://testnet.mintscan.io/noble-testnet/blocks/2682700)
+- **Estimated Date & Time:** Thursday, August 24th, around 11am EST. *(Please note that this is an approximation. Due to varying block times, it's crucial to monitor the chain for the exact upgrade moment.)*
```
Comment (johnletey) [inline testnet/grand-1/chain-upgrades/v4.0.0-alpha2/README.md:9]:
```suggestion
- **Estimated Date & Time:** Thursday, August 24th, around 11am EST. *(Please note that this is an approximation. Due to varying block times, it's crucial to monitor the chain for the exact upgrade time.)*
```

File: testnet/grand-1/chain-upgrades/v4.0.0-alpha2/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+1. **Set Halt Height:** All validators must set the halt height to `2682700`. There is no on chain instruction to halt.
+2. **Use Pre-Built Binaries:** Choose the appropriate binary for your system:
+   - [AMD](./nobled_linux-amd64)
+   - [ARM](./nobled_linux-arm64)
+
+
+### Using Docker?
+
+Pull the latest image with:
```
Comment (johnletey) [inline testnet/grand-1/chain-upgrades/v4.0.0-alpha2/README.md:21]:
```suggestion
Pull the latest image:
```

**Review by johnletey — APPROVED** [review]
Thanks so much for this @danbryan!

Just a few small nits 😄

**Pull request merged**